deploy.ps1

```diff
@@ -223,7 +223,20 @@ $k8sReqs = @(
     #         az extension update --name aks-preview
 
     #         # Apply default policy
-    #         az aks update --resource-group sbd --name sbd --enable-pod-security-policy
+    #         az aks update --resource-group sbd --name sbd --enable-pod-security-polic
```

Comment:
Why change from -policy to -polic?

Reply:
typo, this is commented out and is the next order of business to get integrated now.